main.py

```diff
@@ -19,16 +19,36 @@
     MOCK_ARGUMENT,
     PORT,
     SERVO_PIN,
-    SIMULATION_LOG_NAME,
+    SIMULATION_LOG_PATH,
     UPSIDE_DOWN,
 )
 
+# ANSI escape code for cursor movement:
+MOVE_CURSOR_UP = "\033[F"  # Move the cursor one line up
+
+
+def update_display(airbrakes):
```

Comment:
can we put this in utils?

Reply:
sure. I'm also going to add a "time since sim start" field